packages/constants/src/mcf.ts

```diff
@@ -438,3 +440,700 @@ export const SSEC_EQA_LIST: ISsecEqa[] = [
+  'with',
+  "won't",
+  'would',
+  "wouldn't",
+  'you',
+  "you'd",
+  "you'll",
+  "you're",
+  "you've",
+  'your',
+  'yours',
+  'yourself',
+  'yourselves',
+];
+
+export const removeStopWords = (str = '') => {
+  const strArr = str.split(' ');
+  const sanitisedArr = remove(strArr, (word) => {
+    return !STOP_WORDS.includes(word);
+  });
+  return sanitisedArr.join(' ');
+};
+
+export const removeWordsInBracket = (str = '') => {
+  const re = /\(.*?\)/gi;
+  return str.replace(re, '');
+};
+
+export const removePunctuations = (str = '') => {
+  // '-' is ignored in this case
+  const re = /[~`!@#$%^&*(){}\[\];:"'<,.>?\/\\|_+=]/g;
+  return str.replace(re, '');
+};
+
+export const removeExcessWhitespaces = (str = '') => {
+  const re = /\s+/g;
+  return str.replace(re, ' ');
+};
+
+export const removeRepeatedHyphens = (str = '') => {
+  const re = /\-+/g;
+  return str.replace(re, '-');
+};
+
+export const joinWords = (str = '') => {
+  const re = /\s/g;
+  return str.trim().replace(re, '-');
+};
+
+export const cleanWord = (str = '') => {
+  const currentStr = `${str.toLowerCase()}`;
+  return flow(
+    removeWordsInBracket,
+    removeStopWords,
+    removePunctuations,
+    removeExcessWhitespaces,
+    joinWords,
+    removeRepeatedHyphens,
+    encodeURIComponent,
+  )(currentStr);
+};
+
+export const pathToJobId = (path) => {
+  // Sanitize path to remove last character if last character is `/`
+  const parsedPath = path.substr(-1) === '/' ? path.slice(0, -1) : path;
+  const regexExp = /job\/(?:.*-)?(.*)/i;
+  const groups = regexExp.exec(parsedPath);
+  // length - 1 is to find the uuid, since it will always be the last one is in the group
+  // split('/') will handle the case where the url end with `/apply` in the case of job application
+  const jobId = groups && groups[groups.length - 1] ? groups[groups.length - 1].split('/') : null;
+  // same apply here as well
+  return jobId && (jobId[jobId.length - 1] === 'apply' ? jobId[jobId.length - 2] : jobId[jobId.length - 1]);
+};
+
+export const isJobApplicationPath = (path) => {
+  const regexExp = /\/apply\/?$/;
+  const groups = regexExp.exec(path);
+  return groups ? true : false;
+};
+
+export const pathToJobAlertChecksum = (path) => {
+  const regexExp = /jobalert\/remove\/(?:.*-)?(.*$)/i;
+  const checksum = regexExp.exec(path);
+  return checksum && checksum[1] ? checksum[1] : null;
+};
+
+export const getCategoryByLabel = (label) => Object.keys(CATEGORY).find((category) => CATEGORY[category].label === label);
+
+
+function Key(id: string, label: string, value: string, url: string) {
+  return {id, label, value: typeof value === 'undefined' ? id : value, url};
+}
+
+function  CategoryKey(id: string, label: string, value: string, url: string, metaDescription?: string, pageTitle?: string, description?: string) {
+  return {
+    ...Key(id, label, value, url),
+    description,
+    metaDescription,
+    pageTitle,
+  };
+}
+
+export const CATEGORY_DESCRIPTION = {
```

Comment:
Same thing here, we already declared these in a separate file, is there a reason why they must be declared here again?

Reply:
removed unused file

Reply:
We might also want to put this at the bottom, since it's a large chunk to scroll through, and there are more important constants and functions below :)

Reply:
updated